environment.yaml

```diff
@@ -12,3 +12,8 @@ dependencies:
   - openssl
   - ripgrep
   - poetry=1.1
+  - mykrobe
+  - rasusa
+  - samtools
+  - gsl
```

Comment:
What is gsl?

Reply:
it is a dependency of `bcftools` that I have no idea why it was not getting installed. When running `bcftools` I was getting `bcftools: error while loading shared libraries: libgsl.so.25: cannot open shared object file: No such file or directory` . I tried conda uninstalling, installing again, reinstalling, etc, all with the same error. What fixed for me was explicitly installing `gsl`. Once we are done with the code, I will review the tooling and check if this line is still necessary or superfluous (I'd guess superfluous, as `gsl` has to be a dependency of `bcftools` in `conda`, but who knows...)

Reply:
If you pin bcftools to 1.13 this should go away maybe? This sounds weird and buggy

Reply:
It did, thanks! Done in https://github.com/mbhall88/tbpore/pull/8/commits/d4578bbe9214dd8b9379f269e4739575ba1b62ed